packages/babel-cli/src/babel/options.js

```diff
@@ -161,6 +161,11 @@ commander.option(
   "Delete the out directory before compilation.",
 );
 
+commander.option(
+  "--include-ignore",
+  "Copy the ignored files to destination",
```

Comment:
To match to `--include-dotfiles`,

```suggestion
  "Include ignored files when compiling and copying non-compilable files.",
```